schemas/isd.json

```diff
@@ -0,0 +1,242 @@
+{
+   "$schema": "http://json-schema.org/draft-07/schema#",
+   "title": "Instrument Specification Definition",
+   "description": "The Instrument Specification Definition for Astro CSM",
+   "type": "object",
+
+   "properties": {
+       "BORESIGHT": {
+	   "description": "Boresight (x,y,z) vector",
+           "type": "array",
+	    "items": {
+                 "type": "number",
+		 "description": "Component of the boresight vector",
+		 "minItems": 3,
+                 "maxItems": 3
+            }
+       },
```

Comment:
What is this for? The detector center and focal2pixel parameters cover all of the math needed to account for the boresight.

Reply:
I was going to initially answer that it was _used_ by the `UsgsAstroFrameSensorModel`, but you're right, it's not. It's _required_ by it, but not actually used by it, so this looks like it can be eliminated.